README.md

```diff
@@ -10,10 +10,10 @@ with ECR.
 
 ## Images
 
-We publish two images. One has `gcloud` CLI bundled, the other does not.
+We publish two images. The "normal" has `aws` cli v2 included, the other has `gcloud`.
```

Comment:
I'm not sure I'd use the word "normal" here. It could imply to people that gcloud is abnormal :smile: 

Maybe explicitly mention the image names / tags? So something like: the `latest` tag includes the AWS cli v2, and the `gcp-latest` tag includes gcloud.

Reply:
good idea. yeah, "normal" is a bit hungarianish expression in this context :D

Reply:
I think "default" would be another good option, because it doesn't have any implication of one being better than the other.